java/code/src/com/redhat/rhn/frontend/strings/java/StringResource_en_US.xml

```diff
@@ -735,8 +735,14 @@
           <context context-type="sourcefile">Navigation Menu</context>
         </context-group>
       </trans-unit>
-      <trans-unit id="Proxy Release Notes">
-        <source>Proxy Release Notes</source>
+      <trans-unit id="SUSE Manager Proxy">
+        <source>@@PRODUCT_NAME@@ Proxy</source>
+        <context-group name="ctx">
+          <context context-type="sourcefile">Navigation Menu</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="SUSE Manager Server">
```

Comment:
It's only in the code, but maybe we can use more generic ids here:
`product_server`, `product_proxy` or `release_notes_server`, `release_notes_proxy`

Reply:
changed